frontend/src/actions/cartActions.js

```diff
@@ -0,0 +1,25 @@
+import axios from "axios";
+import { ADD_TO_CART, REMOVE_ITEM_CART } from "../constants/cartConstants";
+
+export const addItemToCart = async (id, dispatch) => {
+  const { data } = await axios.get(`http://localhost:8000/api/product/${id}`);
+  const productData = {
+    id: data.product._id,
+    name: data.product.name,
+    price: data.product.price,
+    image: data.product.image,
+    stock: data.product.stock,
+  };
+
+  dispatch({
+    type: ADD_TO_CART,
+    payload: productData,
+  });
+};
+
+export const removeItemFromCart = async (id, dispatch) => {
```

Comment:
I am not sending the request for the delete cart's item because I am not managing the cart in the backend.  I am managing cart only on frontend.  I will manage the orders only.